anaconda.py

```diff
@@ -320,7 +319,7 @@ def start_debugger(signum, frame):
     # see if we're on s390x and if we've got an ssh connection
     uname = os.uname()
     if uname[4] == 's390x':
-        if 'TMUX' not in os.environ and 'ks' not in flags.cmdline and not conf.target.is_image:
+        if 'TMUX' not in os.environ and 'ks' not in flags.cmdline and conf.target.is_hardware:
```

Comment:
Is this really the same. What about dir install?

Reply:
It was a bug. Actually, you really want to do this only on the boot.iso.

Reply:
In that case could you please separate this to a commit with explanation why this behavior changed?

Reply:
Ok, I will change the behaviour in a new pull request. It shouldn't be part of this change.